dspace-spring-rest/src/main/java/org/dspace/app/rest/model/DirectlyAddressableRestModel.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * A directly addressable REST resource
+ * 
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ *
+ */
+public interface DirectlyAddressableRestModel extends RestModel {
```

Comment:
I'm not entirely sure I understand the meaning of "directly addressable" in this class name/description. I'm not sure I know of a better name for this interface, but it's currently a confusing name to me.

Reply:
We need to distinguish between model that can be only embedded in other model such as the checksum (that have a value and an algorithm) and model that are exposed as Rest Resource (so they have an URL)

Reply:
Could it just be called `RestResourceModel`?  And then we can add in a clearer comment to explain what we mean by that?  We could also ask around for other ideas...or maybe "directly addressable" makes sense to others (but it definitely is a confusing name for me).